tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -363,6 +363,12 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.KafkaCreateConsumerScopeEnabled"/>
         public const string KafkaCreateConsumerScopeEnabled = "DD_TRACE_KAFKA_CREATE_CONSUMER_SCOPE_ENABLED";
 
+        /// <summary>
+        /// Configuration key to enable or disable the updated WCF instrumentation
+        /// </summary>
+        /// <seealso cref="TracerSettings.WcfEnableNewInstrumentation"/>
+        public const string WcfEnableNewInstrumentation = "DD_TRACE_WCF_ENABLE_NEW_INSTRUMENTATION";
```

Comment:
If we actually consider this a "feature flag", I would put into the `FeatureFlags` nested class. 

I'd also prefer something a bit more descriptive, ending in `Enabled` (in case we add _another_ WCF integration 😅), e.g. `DD_TRACE_DELAY_WCF_INSTRUMENTATION_ENABLED`

Reply:
is this following the naming conventions for feature flags?

Reply:
`DD_TRACE_DELAY_WCF_INSTRUMENTATION_ENABLED` would be I think 🙂

Reply:
Good call. I made both changes in e75305ad01e62457aa7ce96009bbe6400439f0f5